src/plots/gl3d/scene.js

```diff
@@ -559,11 +559,13 @@ proto.destroy = function() {
 // for reset camera button in mode bar
 proto.setCameraToDefault = function setCameraToDefault() {
     // as in Gl3d.layoutAttributes
-    this.glplot.camera.lookAt(
+    var lookAtInput = [
         [1.25, 1.25, 1.25],
         [0, 0, 0],
         [0, 0, 1]
-    );
+    ];
+    this.glplot.camera.lookAt.apply(this, lookAtInput);
+    this.graphDiv.emit('plotly_relayout', lookAtInput);
```

Comment:
@etpinard I'm glad to refactor for it; I'll extract the object-to-array logic into a separate function so we can be DRY among these two `emit` places.